src/main/java/com/example/booksstore/exceptions/CustomGlobalExceptionHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.example.booksstore.exceptions;
+
+import java.time.LocalDateTime;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    private static final String TIME = "time";
+    private static final String STATUS = "status";
+    private static final String ERRORS = "errors";
+
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(
```

Comment:
we can also add handling of the `EntityNotFoundException`, `DataProcessingException`(if you have it) and the one which will be thrown at failed ISBN uniqueness validation